orc8r/cloud/go/services/certifier/storage/storage_blobstore_test.go

```diff
@@ -30,10 +30,46 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
+var (
```

Comment:
I don't think hoisting this state up to the pkg level is warranted here. Yes it saves some lines of code per fn, but it contaminates the test_pkg's pkg-level state, so e.g. another _test.go file in this pkg couldn't define its own `someErr`

Let's leave these vars within each fn for now. A little code repetition is fine, and especially in tests it's a typical pattern

If you feel strongly about this, you could create a "test" struct with fields for all these vars. But since we only use them in 2-3 places, it's not really worth over-engineering a test file